recipes/cnsr/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cnsr" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cnsr-{{ version }}.tar.gz
+  sha256: 5869709655591de6cf4608a6ab304caf9b292daebc1e475f62dbdf32d32c1f5a
+
+build:
+  entry_points:
+    - cnsr_copy_notebooks = cnsr.install:copy_notebooks
+    - install_cnsr_launcher_data = cnsr.install:install_cnsr_launcher_data
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=61
+    - pip
+    - click <9
+    - jupyterlab >=3.1,<4
+  run:
+    - python >=3.9
+    - click <9
+    - ipyfilechooser <1
+    - jupyter_app_launcher >=0.1,<1
+    - jupyterlab >=3.1,<4
+    - matplotlib-base <4
+    - mne <2
+    - mne-nirs <1
+    - neurokit2 <1
+    - nilearn <1
+    - numpy <2
+    - openpyxl <4
+    - pandas <3
```

Comment:
In general, upper bounds in conda world should explicitly use the alpha 0 as the upper limit.

So instead of `pkg >=3,<4`, use `pkg >=3,<4.0a0`.